invenio_app_rdm/config.py

```diff
@@ -525,30 +525,45 @@ def files_rest_permission_factory(obj, action):
 """OAI ID fetcher function."""
 
 OAISERVER_METADATA_FORMATS = {
+    "marcxml": {
+        "serializer": "invenio_rdm_records.oai:marcxml_etree",
+        "schema": "https://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd",
+        "namespace": "https://www.loc.gov/standards/marcxml/",
+    },
     "oai_dc": {
-        "serializer": ("invenio_rdm_records.oai:dublincore_etree"),
+        "serializer": "invenio_rdm_records.oai:dublincore_etree",
         "schema": "http://www.openarchives.org/OAI/2.0/oai_dc.xsd",
         "namespace": "http://www.openarchives.org/OAI/2.0/oai_dc/",
     },
+    "dcat": {
+        "serializer": "invenio_rdm_records.oai:dcat_etree",
+        "schema": "http://schema.datacite.org/meta/kernel-4/metadata.xsd",
+        "namespace": "https://www.w3.org/ns/dcat",
+    },
+    "marc21": {
+        "serializer": "invenio_rdm_records.oai:marcxml_etree",
+        "schema": "https://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd",
+        "namespace": "https://www.loc.gov/standards/marcxml/",
+    },
     "datacite": {
         "serializer": "invenio_rdm_records.oai:datacite_etree",
-        "schema": "http://schema.datacite.org/meta/nonexistant/nonexistant.xsd",
-        "namespace": "http://datacite.org/schema/nonexistant",
+        "schema": "http://schema.datacite.org/meta/kernel-4.1/metadata.xsd",
```

Comment:
```suggestion
        "schema": "http://schema.datacite.org/meta/kernel-4.3/metadata.xsd",
```

minor: since we actually serialize DataCite v4.3. Ditto for `datacite4` below.